src/main/java/com/opentable/extension/BodyTransformer.java

```diff
@@ -177,6 +181,20 @@ private CharSequence getValueFromRequestObject(String group, Map requestObject)
         return String.valueOf(tempObject);
     }
 
+    private CharSequence replaceValueFromRequestObject(String group, Map requestObject) {
```

Comment:
getModifiedValueFromRequestObject - it would be good to pass function that actually modifies a value